scripts/us_eia/opendata/process/common.py

```diff
@@ -38,6 +38,11 @@
 def _parse_date(d):
     """Given a date from EIA JSON convert to DC compatible date."""
 
+    try:
+      _ = int(d)
```

Comment:
could d.isnumeric() work here?

Reply:
out of curiosity, what kinds of non-numeric values did you come across? should we add some to the test?

Reply:
I added it as an extra check, but there is actually no non-numeric value so far (ELEC, NG, PET).

Yes, isnumeric() is nicer!

Reply:
My comment about non-numeric is not true. There is 2019Q1 pattern. I'll add a test in upcoming PR.